### DISCUSSION TITLE
Update-DbaServiceAccount vs SQL Configuration manager.

### DISCUSSION THREAD

**Discussion by whoisbruce** [dataplat/dbatools#7761] [Q&A] [answered]
Hi, I'm questioning the validity of using Update-DbaServiceAccount. According to this
https://docs.microsoft.com/en-us/sql/database-engine/configure-windows/configure-windows-service-accounts-and-permissions?view=sql-server-ver15#[…]s SQL Configuration Manager makes other changes. Viewing the code of Update-DbaServiceAccount, it just seems to update only the service. Won't this break things? (Especially the relationship between the account and SSRS keys)?

**Comment by niphlod** [comment, marked as answer]
we use WMI which takes care of not breaking things with SSRS keys.
